src/meltano/cli/__init__.py

```diff
@@ -97,6 +100,8 @@ def _run_cli():
     Raises:
         KeyboardInterrupt: if caught.
     """
+    logger.debug(f"meltano, version : {__version__}")
+    logger.debug(f"operating system : {os.name}")
```

Comment:
This is probably too early. At this point the logging configuration hasn't been loaded. A better place is probably right after https://github.com/meltano/meltano/blob/6c886f89f3dbc61ee1b64d98187c9aaaf7c3d436/src/meltano/cli/cli.py#L113.

Reply:
done